docs/recipes/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "private": true,
   "scripts": {
-    "install-all": "yarn --cwd auth/ && yarn --cwd integration-test/ && yarn --cwd custom-gatherer-puppeteer/",
+    "install-all": "rm -rf /tmp/.junk && rm -f {auth,custom-gatherer-puppeteer,integration-test}/yarn.lock && yarn --cwd auth/ --cache-folder /tmp/.junk && yarn --cwd integration-test/ --cache-folder /tmp/.junk && yarn --cwd custom-gatherer-puppeteer/ --cache-folder /tmp/.junk",
```

Comment:
Do you mind checking if using npm for the docs stuff works as expected instead? Feels like yarn isn't bringing anything to the relationship in this directory

Reply:
Well, at this point I can't recreate the old behavior as it was 1) yarn's cache and 2) I cleared my cache. So we wouldn't be able to detect if this is also an issue in npm.